pages/avatar/[address].tsx

```diff
@@ -61,8 +61,14 @@ const AvatarPage = observer(() => {
   const [tarotStats, setTarotStats] =
     useState<Tarot.TarotStatsForAddress>(null);
 
+  const isOwner =
+    store.wallets.activeAccount?.address === address ||
+    store.wallets.activeAccount?.address === zeitAddress;
+
   const inventory = useInventoryManagement(
-    (store.wallets.getActiveSigner() as ExtSigner) || address,
```

Comment:
Was showing inventory of badges for logged in address with no respect for the url man